Gemfile.lock

```diff
@@ -16,3 +16,6 @@ DEPENDENCIES
   attr_comparable!
   minitest
   rake (>= 0.9)
+
+BUNDLED WITH
+   1.12.5
```

Comment:
I thought we were keeping these comments out. I know they broke older versions of bundler. That might not matter any more though...